README.md

```diff
@@ -1,27 +1,30 @@
-<img src="https://raw.githubusercontent.com/mineiros-io/brand/master/mineiros-vertial-logo-smaller-font.svg" width="200"/>
+[<img src="https://raw.githubusercontent.com/mineiros-io/brand/master/mineiros-primary-logo.svg" width="400"/>](https://www.mineiros.io/?ref=terraform-aws-route53)
 
-[![Maintained by Mineiros.io](https://img.shields.io/badge/maintained%20by-mineiros.io-f32752.svg)](https://mineiros.io/?ref=terraform-aws-route53)
 [![Build Status](https://mineiros.semaphoreci.com/badges/terraform-aws-route53/branches/master.svg?style=shields)](https://mineiros.semaphoreci.com/projects/terraform-aws-route53)
 [![GitHub tag (latest SemVer)](https://img.shields.io/github/v/tag/mineiros-io/terraform-aws-route53.svg?label=latest&sort=semver)](https://github.com/mineiros-io/terraform-aws-route53/releases)
-[![Terraform Version](https://img.shields.io/badge/terraform-~%3E%200.12.20-brightgreen.svg)](https://github.com/hashicorp/terraform/releases)
-[![License](https://img.shields.io/badge/License-Apache%202.0-brightgreen.svg)](https://opensource.org/licenses/Apache-2.0)
+[![license](https://img.shields.io/badge/license-Apache%202.0-brightgreen.svg)](https://opensource.org/licenses/Apache-2.0)
+[![Terraform Version](https://img.shields.io/badge/terraform-~%3E%200.12.20-623CE4.svg)](https://github.com/hashicorp/terraform/releases)
+[<img src="https://img.shields.io/badge/slack-@mineiros--community-f32752.svg?logo=slack">](https://join.slack.com/t/mineiros-community/shared_invite/zt-ehidestg-aLGoIENLVs6tvwJ11w9WGg)
 
 # terraform-aws-route53
+
 A [Terraform](https://www.terraform.io) 0.12 module to create a scalable and highly available
-[Amazon Route53](https://aws.amazon.com/service/route53) Domain Name System (DNS) on
+[Amazon Route53](https://aws.amazon.com/route53/) Domain Name System (DNS) on
 [Amazon Web Services (AWS)](https://aws.amazon.com/).
 
 - [Module Features](#module-features)
 - [Getting Started](#getting-started)
 - [Examples](#examples)
-- [Makefile](#makefile)
+  - [Makefile](#makefile)
```

Comment:
wrong level, wrong location in the file and wrong title..

Reply:
resolved in a7380df